docs/tutorials/tutorial-batch-hadoop.md

```diff
@@ -148,7 +148,7 @@ From the Hadoop container's shell, run the following command to copy the Hadoop
 cp /usr/local/hadoop/etc/hadoop/*.xml /shared/hadoop_xml
 ```
 
-From the host machine, run the following, where {PATH_TO_DRUID} is replaced by the path to the Druid package.
+From the host machine, run the following, where \{PATH_TO_DRUID\} is replaced by the path to the Druid package.
```

Comment:
Does this get interpreted at build or later? I see
```
From the host machine, run the following, where {PATH_TO_DRUID} is replaced by the path to the Druid package.
```

Reply:
No, this is not something that's supposed to be rendered. So it could have been put in code ticks or escaping the {

current live site:
<img width="1019" alt="image" src="https://github.com/user-attachments/assets/79569f91-5363-43fd-975a-4ff87dce48f5" />